partner_test-data_fenix/data/res_company.xml

```diff
@@ -0,0 +1,16 @@
+<odoo>
+    <data noupdate="1">
+        <record id="jack_sons_company" model="res.company">
```

Comment:
jack_sons_company?
Is this test data in some way?

Reply:
Yes, it's the raw data that defines an instance of a the test models. The same with the same with res_users.xml & res_partner.xml. I tried it on a empty DB and it works!